src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star star--active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars__star star--active"></div>
+        <div class="stars__star star--active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--3">
+        <div class="stars__star star--active"></div>
+        <div class="stars__star star--active"></div>
+        <div class="stars__star star--active"></div>
```

Comment:
we don't need these active classes, let's make our code more generic and implement logic only with this 
<img width="829" alt="image" src="https://github.com/mate-academy/layout_stars/assets/70220667/320c02f0-0e6c-4de0-816a-6d716978d323">